sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -389,6 +389,18 @@ object SQLConf {
     .booleanConf
     .createWithDefault(true)
 
+  val STARJOIN_OPTIMIZATION = buildConf("spark.sql.starJoinOptimization")
```

Comment:
I am not entirely sure about the namespace for both these confs. Does it make sense to you to use the `spark.sql.cbo` namespace.